src/styles/components/_specification-block.scss

```diff
@@ -18,10 +23,27 @@ $breakpoint-tablet: 820px;
 
   ul {
     list-style: url("../../../public/teal-decorative-bullet.svg");
+    line-height: 1.5625rem;
+  }
+
+  li {
+    font-weight: 400;
+    letter-spacing: 175%;
+  }
+
+  /* Adjacent sibling selector */
+  li + li {
+    margin-top: 1.5rem;
   }
 
   code {
     color: map.get($colors, "blue");
+    font-family: Oxygen, sans-serif;
+  }
+
+  h3 {
```

Comment:
font-size for `h3` is also just a tad too big on mobile

<img width="1436" alt="image" src="https://user-images.githubusercontent.com/54716846/200096692-c8c12039-4002-43e6-8dac-c7d480b1bc82.png">